genesyscloud/resource_exporter/resource_exporter_custom.go

```diff
@@ -162,9 +162,6 @@ func RuleSetSkillPropertyResolver(configMap map[string]interface{}, exporters ma
 				if exists {
 					exportId = value.Name
 					sanitisedSkillIds = append(sanitisedSkillIds, fmt.Sprintf("${genesyscloud_routing_skill.%s.id}", exportId))
-				} else {
```

Comment:
removing this , will make sure the import to new org will also work. the deleted skills are simply ignored in the skillset references.